gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/PortalManageSites.java

```diff
@@ -339,15 +349,17 @@ public void setEditPermissions(String groupId, String membership) {
 
     info("--Setting edit permission to " + groupId + ", " + membership + "--");
     String[] groups = groupId.split("/");
-    evt.click(ELEMENT_SELECT_PERMISSION_BUTTON);
+    $(byXpath("//*[@id=\"UIPermissionSelector\"]/div[3]/a[2]")).click();
```

Comment:
Shouldn't we update ELEMENT_SELECT_PERMISSION_BUTTON instead ?
Or at least we should create a new locator in GateinLocator.